### DISCUSSION TITLE
[SOLVED] defineModel Vue 3.3.0-beta.2

### DISCUSSION THREAD

**Discussion by angelhdzmultimedia** [vue-macros/vue-macros#350] [Q&A] [answered]
I'm trying the official experimental macro. I tried everything. updating vue to 3.3.0-beta.2, installing 3.3.0-beta.2 of @vue/compiler-sfc, installing latest version of Vite, adding defineModel: true in vue Vite plugin, using overrides for vue and @vue/compiler-sfc.

I first tried this in a Nuxt 3 project and had issues, so I though that switching to a pure Vue 3 project I would get better results.

I really need a hand please.

**Comment by sxzz** [comment]
I don't know what exactly your problem is. Also, `defineModel` is not the scope of Vue Macros. You can ask the detailed issue in https://github.com/vuejs/core/discussions

**Reply by angelhdzmultimedia** [reply]
> I don't know what exactly your problem is

Well, I think I explained it well. I tried the official experimental `defineModel` in a Nuxt and Pure Vue projects and couldn't get it working, even after installing the beta version of the packages mentioned.

I of course already posted in vuejs/core, nuxt, Discord. I wrote here as a last resort, as I've seen you are involved in all this stuff, hoping you could kindly head me in the right direction. 

> Also, defineModel is not the scope of Vue Macros

To be fair, they both achieve the same thing, so it's not out of scope. Probably this package inspired the Vue team to add this feature officially. I'm just trying to try it because people are talking about it on YouTube, Twitter.


I didn't deserve this cold response. Thanks anyways.

**Reply by sxzz** [reply]
Sorry about my answer. I'm glad to help you to solve the problem, but generally there will be a more friendly answer (for me) if you provide more detail, like reproduction. I cannot get any useful information for your content TBH.

**Comment by angelhdzmultimedia** [comment, marked as answer]
### Solved
For anyone (or my forgetful future self):

`nuxt.config.ts`
```ts
export default defineNuxtConfig({
  vite: {
    vue: {
      script: {
        defineModel: true,
      },
    },
```
